target/linux/samsung/dts/TQ210.dts

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
If you are planning to add more boards with this SoC on it it could be better add this as a patch to the kernel tree.